joomla-projects/joomla-es6
# Remove inline Javascript for administrator/components/com_languages/tmpl/language/edit.php

Pull Request for Issue #55.

### Summary of Changes
Remove Inline JavaScript, jQuery to JavaScript and es6


### Testing Instructions
If you like to test open Extensions | Language | Content Language in Backend and select one language. In this view change the language in the select and see, that the flagimage is changed as well.
